lib/utils/patterns.ts

```diff
@@ -276,11 +276,11 @@ export function printErrorMessage(message: string) {
 	console.error(chalk.red(`\n${messages.getHelp}\n`));
 }
 
-export function exitWithExpectedError(message: string | Error) {
+export function exitWithExpectedError(message: string | Error): never {
 	if (message instanceof Error) {
 		({ message } = message);
 	}
 
 	printErrorMessage(message);
-	process.exit(1);
+	return process.exit(1);
```

Comment:
I'm not sure about the `return` but don't have a strong opinion either :+1:

Reply:
It's required to make the `never` typings work properly, otherwise TS doesn't know what this function will never return.